rmc/utils/constraint.py

```diff
@@ -719,7 +794,7 @@ def search_for_break(
     )
     if save_chisq_ht:
         all_loci_chisq_ht_path = (
-            f"{MODEL_PREFIX}/{CURRENT_GNOMAD_VERSION}/{freeze}/constraint_prep.ht"
+            f"{MODEL_PREFIX}/{CURRENT_GNOMAD_VERSION}/{freeze}/all_loci_chisq.ht"
```

Comment:
I moved this out of temp because this is a useful table to have (particularly if we decide to experiment with more p-value thresholds again), so I didn't think it should go into temp; I'm hoping that we'll be a bit more mindful of what we store in temp and for how long in the next rerun.

how about `CONSTRAINT_PREFIX`? it's not quite an output file, but you're right that it isn't really related to the mutational models

Reply:
Sounds good to me!